src/test/endorsement-chain.spec.js

```diff
@@ -5,7 +5,12 @@ fixture("Endorsement Chain Rendering").page`http://localhost:3000`;
 
 const ViewEndorsementChainButton = Selector("#endorsement-chain-button");
 const EndorsementChainTitle = Selector(".endorsement-chain-title");
```

Comment:
i think this can be better, use `withText` as well, because the assertion is testing for count which most likely return true but it does not guarantee the correct text exists

Reply:
ok